packages/storybook/src/nimble/number-field/number-field.stories.ts

```diff
@@ -78,6 +80,12 @@ const metadata: Meta<NumberFieldArgs> = {
                 'The number displayed in the number field. Note that the property value is not synced to an attribute.',
             table: { category: apiCategory.nonAttributeProperties }
         },
+        placeholder: {
```

Comment:
Can you make the label and placeholder independently configurable? Similar to the text field story

Reply:
oh you just pushed that, nice

Reply:
Yep, was an oversight.